hal/src/electron/cellular_internal.cpp

```diff
@@ -54,9 +54,11 @@ cellular_result_t cellular_signal_impl(CellularSignalHal* signal, cellular_signa
             signalext->rat = NET_ACCESS_TECHNOLOGY_UTRAN;
             break;
         case ACT_LTE:
+            signalext->rat = NET_ACCESS_TECHNOLOGY_LTE;
```

Comment:
These changes go along with the change above to move Cat-NB1 down into NET_ACCESS_TECHNOLOGY_LTE_IOT

Reply:
:+1: